spec/source/locales-currencies-tz.html

```diff
@@ -0,0 +1,163 @@
+    </p>
+
+
+    <emu-clause id="sec-unicode-locale-extension-sequences">
+      <h1>Unicode Locale Extension Sequences</h1>
+
+      <p>
+        This standard uses the term "Unicode locale extension sequence" for any substring of a language tag that is not part of a private use subtag sequence, starts with a separator *"-"* and the singleton *"u"*, and includes the maximum sequence of following non-singleton subtags and their preceding *"-"* separators.
+      </p>
+    </emu-clause>
+
+    <emu-clause id="sec-isstructurallyvalidlanguagetag">
+      <h1>IsStructurallyValidLanguageTag (locale)</h1>
+
+      <p>
+        The IsStructurallyValidLanguageTag abstract operation verifies that the _locale_ argument (which must be a String value)
+      </p>
+
+      <ul>
+        <li>represents a well-formed BCP 47 language tag as specified in RFC 5646 section 2.1, or successor,</li>
+        <li>does not include duplicate variant subtags, and</li>
+        <li>does not include duplicate singleton subtags.</li>
+      </ul>
+
+      <p>
+        The abstract operation returns true if _locale_ can be generated from the ABNF grammar in section 2.1 of the RFC, starting with Language-Tag, and does not contain duplicate variant or singleton subtags (other than as a private use subtag). It returns false otherwise. Terminal value characters in the grammar are interpreted as the Unicode equivalents of the ASCII octet values given.
+      </p>
+    </emu-clause>
+
+    <emu-clause id="sec-canonicalizelanguagetag">
+      <h1>CanonicalizeLanguageTag (locale)</h1>
+
+      <p>
+        The CanonicalizeLanguageTag abstract operation returns the canonical and case-regularized form of the _locale_ argument (which must be a String value that is a structurally valid BCP 47 language tag as verified by the IsStructurallyValidLanguageTag abstract operation). It takes the steps specified in RFC 5646 section 4.5, or successor, to bring the language tag into canonical form, and to regularize the case of the subtags, but does not take the steps to bring a language tag into "extlang form" and to reorder variant subtags.
+      </p>
+
+      <p>
+        The specifications for extensions to BCP 47 language tags, such as RFC 6067, may include canonicalization rules for the extension subtag sequences they define that go beyond the canonicalization rules of RFC 5646 section 4.5. Implementations are allowed, but not required, to apply these additional rules.
+      </p>
+    </emu-clause>
+
+    <emu-clause id="sec-defaultlocale">
+      <h1>DefaultLocale ()</h1>
+
+      <p>
+        The DefaultLocale abstract operation returns a String value representing the structurally valid (<a href="#sec-isstructurallyvalidlanguagetag">6.2.2</a>) and  canonicalized (<a href="#sec-canonicalizelanguagetag">6.2.3</a>) BCP 47 language tag for the host environment’s current locale.
```

Comment:
for now, cross references to other sections have to be manually defined, once @bterlson adds the new element for that (https://github.com/bterlson/ecmarkup/issues/23) we can change it all together.